sensor_msgs/Image robot_view
---
string caption_result
